### PR TITLE
Refactoring for Readability and Extensibility

### PR DESCRIPTION
**Pull request by penwoodjon**
```
admin@example.com
45.128.134.14
10.10.240.126
84.2.64.5
INC0010004
```

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -4,22 +4,221 @@ const {
...
+  })
+)(customTypes);
+
+
+// TODO: Things to Add
+// mapping layout and property maps by type
+/**
+ * TODO:
+ * change do comments and variable names to make sense
+ *
```
Comment (ntsmith7) [inline src/constants.js]: Not sure if these were left here intentianlly.

File: src/constants.js [hunk 2]
```diff
@@ -4,22 +4,221 @@ const {
...
+        )
+      : type;
+
+const getQueryFunctionType = createTypeCheckFunction(customEntityTypesWithCustomQueryFunction);
+const getSummaryType = createTypeCheckFunction(customEntityTypesWithCustomTagCreation);
+
+const asdf = {
+  ...merge(DEFAULT_STANDARD_MAPPING, ENTITY_TYPE_BASED_MAPPING),
+  ...merge(DEFAULT_CUSTOM_MAPPING, CUSTOM_ENTITY_TYPE_BASED_MAPPING)
+};
```
Comment (ntsmith7) [inline src/constants.js]: This variable name could be changed.

File: src/constants.js (hunk 1 above)
Comment (penwoodjon) [reply src/constants.js]: This comment is on a the constants.js file from 8 commits from when the card was put in PR.

File: src/constants.js (hunk 2 above)
Comment (penwoodjon) [reply src/constants.js]: This comment is on a the constants.js file from 8 commits from when the card was put in PR.

**Pull request merged**